Feature/ONG4-125/Footer

gabrielpindo opened the pull request:
Se realizó el componente <Footer/> que muestra información de la ONG "Somos más" de forma dinámica. 

![Footer1](https://user-images.githubusercontent.com/69809704/202538414-6e1d87c2-0d2e-45ce-b0eb-b4c616a3acbf.png)
![Footer2](https://user-images.githubusercontent.com/69809704/202538368-a0d3f0df-8472-4885-852c-c36825092147.png)
![Footer3](https://user-images.githubusercontent.com/69809704/202538376-2a512af0-3992-4315-b434-1060e37f8c94.png)

gabrielpindo committed:
Creación del Footer

Se creó el footer de la página web, consumiendo la información desde la Api.

gabrielpindo committed:
Realización del componente Footer, pié de página de la ONG-Somos mas

gabrielpindo committed:
Merge branch 'dev' of https://github.com/alkemyTech/React-SXXI-T4 into feature/ONG4-125/Footer

gabrielpindo committed:
Update: <Footer/> eliminación de console.log()

SoraBeat reviewed (approved):
Muy buen trabajo!! en lo personal diria de utilizar ruta absoluta aqui. Tambien como buena practica utilizar la etiqueta footer en vez de div como contenedor principal
![image](https://user-images.githubusercontent.com/30334118/203127037-f4a4b773-87bc-4705-b0a3-6a201ed2e9b3.png)

milandre commented on src/Services/Organization/ApiService.js:
```diff
@@ -12,6 +12,5 @@ export const getOrganization = setData => {
 export const putOrganization = (values, id) => {
 	axios
 		.put(`https://ongapi.alkemy.org/api/organization/${id}`, values)
-		.then(res => console.log(res))
```
No borrar

milandre commented on src/Components/Layout/components/Footer.jsx:
```diff
@@ -1,10 +1,80 @@
+				<FormNewsletter />
+				<div className="flex flex-col md:flex-row items-center justify-around w-full p-4">
+					<h2>
+						{(dataOrg.cellphone !== null) && "Celular: " + dataOrg.cellphone}
```
Acá pudieses usar el !! (not-not) (`!!dataOrg.cellphone`)

Más info: https://love2dev.com/blog/javascript-not-operator/

milandre commented on src/Components/Layout/components/Footer.jsx:
```diff
@@ -1,10 +1,80 @@
+						{(dataOrg.cellphone !== null) && "Celular: " + dataOrg.cellphone}
+					</h2>
+					<h2>
+						{(dataOrg.address !== null) && "Dirección: " + dataOrg.address}
```
Acá pudieses usar el !! (not-not) (`!!dataOrg.address`)

Más info: https://love2dev.com/blog/javascript-not-operator/

milandre reviewed (changes requested):
Te dejo unos comentarios y @SoraBeat tambien

gabrielpindo committed:
Correcciones requeridas del Footer

Se utilizó el not-not (!!dataOrg.cellphone) al igual que con la dirección.
Se restauró el console.log() del Get.

gabrielpindo committed:
Update Footer.jsx

Se utilizan Rutas absolutas.

gabrielpindo commented:
Cambios realizados.

gabrielpindo committed:
Merge branch 'dev' of https://github.com/alkemyTech/React-SXXI-T4 into feature/ONG4-125/Footer
